### PR TITLE
Add usb emulation for hostdev testing

### PR DESCRIPTION
**Pull request by victortoso**
This is possible since:
  https://github.com/kubevirt/kubevirtci/pull/996

The follow CI would benefit from it:
  https://github.com/kubevirt/kubevirt/pull/10015

### CODE REVIEW

**Comment by victortoso** [comment]
```
14:31:09:   [FAILED] Timed out after 95.106s.
14:31:09:   Expected
14:31:09:       <string>: virt-controller-bf5b79f8d-tm2cz
14:31:09:   to equal
14:31:09:       <string>: virt-controller-bf5b79f8d-7c68f
14:31:09:   In [It] at: tests/infra_test.go:1203 @ 07/06/23 14:31:01.739
14:31:09: 
14:31:09:   Full Stack Trace
14:31:09:     kubevirt.io/kubevirt/tests_test.glob..func9.11.1.1()
14:31:09:     	tests/infra_test.go:1203 +0x1e2 
```

/rehearse rehearsal-pull-kubevirt-e2e-k8s-1.27-sig-compute

**Comment by brianmcarey** [comment]
/cc @xpivarc 

I don't see any usb disks created in the rehearsal job logs - are we missing something here?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -1645,6 +1645,8 @@ presubmits:
           value: "true"
         - name: KUBEVIRT_WITH_ETC_IN_MEMORY
           value: "true"
+        - name: KUBEVIRT_PROVIDER_EXTRA_ARGS
+          value: --usb 30 --usb 40
```
[the remark this answers is not in the document]
Comment (victortoso) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1649]: Ah! Thanks, let's see if that fixes it!

**Comment by xpivarc** [comment]
/rehearse ?
I could successfully run it with the main but used a newer image. Let me fallback to one used by these jobs

**Comment by victortoso** [comment]
/rehearse pull-kubevirt-e2e-k8s-1.26-sig-compute

```
15:19:15: 15:19:15: WARNING: Download from https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.ContentLengthMismatchException Bytes read 74710480 but wanted 202599005
15:19:15: 15:19:15: ERROR: An error occurred during the fetch of repository 'remotejdk11_linux':
15:19:15: 15:19:15:    Traceback (most recent call last): 
...
15:19:15: 15:19:15: Error in download_and_extract: java.io.IOException: Error downloading [https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz] to /tmp/cache/bazel/18316b1300bb8985bc913139d5cc6323/external/remotejdk11_linux/temp8656814839194484506/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz: Bytes read 74710480 but wanted 202599005
15:19:15: 15:19:15: ERROR: /tmp/cache/bazel/18316b1300bb8985bc913139d5cc6323/external/bazel_tools/tools/jdk/BUILD:29:19: @bazel_tools//tools/jdk:current_java_runtime depends on @remotejdk11_linux//:jdk in repository @remotejdk11_linux which failed to fetch. no such package '@remotejdk11_linux//': java.io.IOException: Error downloading [https://mirror.bazel.build/openjdk/azul-zulu11.50.19-ca-jdk11.0.12/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz] to /tmp/cache/bazel/18316b1300bb8985bc913139d5cc6323/external/remotejdk11_linux/temp8656814839194484506/zulu11.50.19-ca-jdk11.0.12-linux_x64.tar.gz: Bytes read 74710480 but wanted 202599005
15:19:15: 15:19:15: ERROR: Analysis of target '//tests:go_default_test' failed; build aborted: 
15:19:15: 15:19:15: INFO: Elapsed time: 7.107s 
```

**Pull request merged**